packages/docs/page-config/ui-elements/toast/api-description.ts

```diff
@@ -0,0 +1,23 @@
+export default {
```

Comment:
We can use define function here similar to `defineManualApi`. That way we'll have proper typing.